src/auto/mount_operation.rs

```diff
@@ -0,0 +1,141 @@
+// This file was generated by gir (8080733) from gir-files (469db10)
+// DO NOT EDIT
+
+use Window;
+use ffi;
+use gdk;
+use glib;
+use glib::StaticType;
+use glib::Value;
+use glib::object::Downcast;
+use glib::object::IsA;
+use glib::signal::SignalHandlerId;
+use glib::signal::connect;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::boxed::Box as Box_;
+use std::mem;
+use std::mem::transmute;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct MountOperation(Object<ffi::GtkMountOperation, ffi::GtkMountOperationClass>);
+
+    match fn {
+        get_type => || ffi::gtk_mount_operation_get_type(),
+    }
+}
+
+impl MountOperation {
+    //pub fn new<'a, P: IsA<Window> + 'a, Q: Into<Option<&'a P>>>(parent: Q) -> MountOperation {
```

Comment:
Strange why this not generated, without `new` `MountOperation` unusable

Reply:
Bug in Gir/Gtk. I'll maybe file a bug. Don't know yet...